modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -17,11 +17,17 @@ You can also use the Agent-based Installer to generate or accept Zero Touch Prov
 * You read the documentation on xref:../../installing/installing-preparing.html#installing-preparing-selecting-cluster-type[Selecting a cluster installation method and preparing it for users].
 * You have read the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#about-the-agent-based-installer[Preparing to install with the Agent-based Installer] documentation.
 * You downloaded the xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent-retrieve_installing-with-agent-based-installer[Agent-Based Installer] and the command-line interface (CLI) from Red Hat’s Hybrid Cloud Console.
-* For a disconnected environment, you created a container image registry, such as Red Hat Quay. See xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-introduction_installing-mirroring-creating-registry[Mirror registry for Red Hat OpenShift introduction].
 * You have logged into the {product-title} with administrator privileges. 
 
 .Procedure
 
+. For a disconnected environment, you mirrored the {product-mirror-registry} to your local container image registry. See xref:../../installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the {product-title} image repository].
```

Comment:
Hi @bfournie and @rwsu . Ju wants me to [tag](https://redhat-internal.slack.com/archives/C04RBMZCBGW/p1709644205791869?thread_ts=1709604598.725869&cid=C04RBMZCBGW) you here to check that this update is correct based on feedback from this [document](https://docs.google.com/document/d/1JEi3YgFWp04ZqEo46tIhsgH4ZHzyKVLJj_2FlFUE9a4/edit). Would you be OK to review?